File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int j=0,k=0;
        string s="";
        
        int n=word1.size(),m=word2.size();
        
        while(j<n || k<m){
            if(j<n){
                s+=word1[j];
                j++;
            }
            if(k<m){
                s+=word2[k];
                k++;
            }
        }
        
        return s;
        
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        n=len(word1)
        m=len(word2)
        j=0
        k=0
        s="";
        
        while j<n or k<m:
            if j<n:
                s+=word1[j]
                j=j+1
            if k<m:
                s+=word2[k]
                k=k+1
        return s